src/pages/workspaces/migration/WorkspaceItem.test.ts

```diff
@@ -1,28 +1,84 @@
+import { DeepPartial } from '@terra-ui-packages/core-utils';
 import { screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { axe } from 'jest-axe';
 import { h } from 'react-hyperscript-helpers';
+import { Ajax } from 'src/libs/ajax';
 import { MigrationStep, WorkspaceMigrationInfo } from 'src/pages/workspaces/migration/migration-utils';
 import { WorkspaceItem } from 'src/pages/workspaces/migration/WorkspaceItem';
-import { renderWithAppContexts as render } from 'src/testing/test-utils';
+import { asMockedFn, renderWithAppContexts as render } from 'src/testing/test-utils';
+
+type AjaxContract = ReturnType<typeof Ajax>;
+type AjaxWorkspacesContract = AjaxContract['Workspaces'];
+jest.mock('src/libs/ajax');
 
 describe('WorkspaceItem', () => {
-  it('shows the workspace name for an unscheduled workspace', async () => {
+  const unscheduledWorkspace: WorkspaceMigrationInfo = {
+    namespace: 'billing project',
+    name: 'workspace name',
+    migrationStep: 'Unscheduled',
+  };
+  const mockGetBucketUsage = jest.fn();
+
+  beforeEach(() => {
+    const mockWorkspaces: DeepPartial<AjaxWorkspacesContract> = {
+      workspace: () => ({
+        bucketUsage: mockGetBucketUsage,
```

Comment:
Added default mocking so we can verify that we don't fetch bucket usage if a workspace has already been scheduled for migration.